README.md

```diff
@@ -69,7 +69,11 @@ Run `letsencrypt_webfaction init` to generate a registration cert and the config
 
 Now, you are ready to run `letsencrypt_webfaction run` from your SSH session to get certificates. Note that by default the config file `letsencrypt_webfaction.toml` is pointed at the LetsEncrypt staging endpoint (the line that says: `endpoint = "https://acme-staging.api.letsencrypt.org/"`); meaning you will only get "test" certificates installed while using the stage endpoint. To issue live certificates you will need to comment out default line, and uncomment the production endpoint line (the line that says: `endpoint = "https://acme-v01.api.letsencrypt.org/" # Production`).
 
-After you run this command, you will see new certificates in the webfaction admin panel (under `Domains / Websites`, then `SSL certificates`), with the names you have provided (ie, the `name = ` variable in the `letsencrypt_webfaction.toml` config file). You need to change your application to point to this certificate after the certificate has been issued. Future runs of this command will update the existing certificate entry and not require a change in the admin. In an SSH session, you can see the certficates generated in `/home/yourusername/certificates/`.
+After you run this command, you will see new certificates in the webfaction admin panel (under `Domains / Websites`, then `SSL certificates`), with the names you have provided (ie, the `name = ` variable in the `letsencrypt_webfaction.toml` config file). You need to change your application to point to this certificate after the certificate has been issued. This change will take some time to propagate through DNS before you will be able to successfully connect with the `https` version of your site. After a short period, webfaction will copy your generated certficates into the directory `/home/yourusername/certificates/`. Future runs of this command will update the existing certificate entry and not require a change in the admin panel.
```

Comment:
One technical note: The delay is not (to the best of my knowledge) a result of DNS. I don't think any A records, CNAMEs, et cetera change as a result of certificate deployment. I think the delay is because WebFaction applies the configuration changes to their frontend servers asynchronously (I don't know, but I can imagine a job running every minute or two and batch applying all the config). All traffic to our sites proxy through a set of frontend Nginx servers that WF manages, which is also where TLS is terminated. It'd probably be more accurate to say "This change will take some time for Webfaction to deploy before you will be able to successfully connect…"

Reply:
I don't know, feel free to change as you wish. I wrote this because once I change the location of certs Webfaction said changes might take time to propagate through DNS and before they did I'd get errors saying in my browser saying something about receiving the webfaction cert... The creation of '~/certificates/' is related to their scripting timing...